### DISCUSSION TITLE
You are attempting to export "metadata" from a component marked with "use client"

### DISCUSSION THREAD

**Discussion by jkiss** [vercel/next.js#68708] [Help]
### Summary

first of all: I;m NOT use "use client" directive. I found it appeared when I use Group Router, does it a BUG? 

if I delete Group Router(stories), everything goes well.

![image](https://github.com/user-attachments/assets/31d73fbb-04be-4f36-9109-6a81dba1c814)

**Comment by Finalet** [comment]
Same issue, I am **not** using "use client" and trying to export metadata object, but getting the `You are attempting to export "metadata" from a component marked with "use client", which is disallowed.` error.

The import trace shows a child component imported from a different file, which, indeed, is a "use client". But, isn't that how things should be?

**Comment by joaobose** [comment]
I'm having the same issue.

**Reply by icyJoseph** [reply]
Do you have a repository to look at?

I recently helped someone https://github.com/vercel/next.js/discussions/82434 - their issue was importing something else from layout/page into a `use client` component.
